compiler/src/main/kotlin/se/ansman/kotshi/AdaptersProcessingStep.kt

```diff
@@ -91,6 +92,15 @@ class AdaptersProcessingStep(
                 ?.map { it as TypeVariableName }
                 ?: emptyList()
 
+        nameAllocator.newName("OPTIONS")
+        nameAllocator.newName("value")
+        nameAllocator.newName("writer")
+        nameAllocator.newName("reader")
+        nameAllocator.newName("types")
+        nameAllocator.newName("moshi")
+        nameAllocator.newName("stringBuilder")
+        (0 until adapterKeys.size).forEach { nameAllocator.newName(generateAdapterFieldName(it)) }
```

Comment:
Perhaps we should actually use the return values here but all of them are valid java identifiers and it just seemed easier to reserve them.